### PR TITLE
Use open-sourced emulator, enable tests on CI

### PR DESCRIPTION
**Pull request by turbolent**

### CODE REVIEW

File: lib/go/test/flow_idtable_staking_test.go [hunk 1]
```diff
@@ -76,16 +75,16 @@ func TestIDTable(t *testing.T) {
 		false,
 	)
 
-	var idTableAddress sdk.Address
+	var idTableAddress flow.Address
```
Comment (joshuahannan) [inline lib/go/test/flow_idtable_staking_test.go:78]: What is the difference between using `sdk.Address` and `flow.Address`? I was using `flow.Address` before, but had to change to `sdk.Address` to get some of the event parsing to work. It is great if this works though!

**Review by joshuahannan — COMMENTED** [review]
Did all the tests pass? Some of the deployment transactions weren't using the proper multiple contracts per account logic.

**Comment by turbolent** [comment]
@joshuahannan Yep, I'm still updating them

File: lib/go/test/flow_idtable_staking_test.go (hunk 1 above)
Comment (turbolent) [reply lib/go/test/flow_idtable_staking_test.go:78]: There's none. There were two imports for the SDK, one plain (package name `flow`), and another one with alias `sdk`. I replace with one single import without alias, might avoid confusion. Can also easily revert!

Comment (joshuahannan) [reply lib/go/test/flow_idtable_staking_test.go:79]: D'oh! No this is fine. I was just confused. Thanks for clarifying! 😅

**Comment by turbolent** [comment]
@joshuahannan it's ready now

**Review by joshuahannan — APPROVED** [review]
Looks good! Thanks for making these updates. 😃

**Pull request merged**